binr/rafind2/rafind2.c

```diff
@@ -221,6 +272,9 @@ int main(int argc, char **argv) {
 		case 'n':
 			nonstop = 1;
 			break;
+		case 'd':
```

Comment:
wat? describe? wasnt that about recursivity?

Reply:
I actually put it in a separate commit and I guess it is ok to drop this one, but I thought it makes sense to put the filename before search results if searching recursively through multiple files